.github/workflows/aws-docker-deploy.yml

```diff
@@ -3,11 +3,25 @@ name: AWS Docker Deploy
 on:
   workflow_call:
     inputs:
+      aws-region:
+        required: true
+        type: string
+      aws-user-pool-id:
+        required: true
+        type: string
+      aws-user-pool-web-client-id:
+        required: true
+        type: string
+      frontend-client-api-url:
+        required: true
+        type: string
+      client-environment-file-path:
+        required: true
+        type: string
```

Comment:
not sure the required: true is needed to be true, as for the backend we won't need it.. :/

Reply:
Je sais pas à quel point ça serait facile / fesable, mais dans l'esprit de garder l'action "niaiseuse", j'aurais tendance à préférer qu'il n'y ait qu'un seul input qui soit le contenu du `.env` à écrire. Ça éviterait aussi d'ajouter des paramètres dont une seule application aurait besoin.